doc/developers/code-style.md

```
@@ -0,0 +1,690 @@
+return $config
+    ->setRules([
+        '@PSR12' => true,
+    ])
+    ->setFinder($finder)
+    ->setRiskyAllowed(false)
+    ->setUsingCache(true)
+    ->setCacheFile(__DIR__ . '/.php-cs-fixer.cache');
+```
+
+**コマンド**:
+```bash
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)